code/__DEFINES/colors.dm

```diff
@@ -1,38 +1,39 @@
-// This is eventualy for wjohn to add more color standardization stuff like I keep asking him >:(
+// This is eventually for wjohn to add more color standardization stuff like I keep asking him >:(
 
 #define COLOR_INPUT_DISABLED "#F0F0F0"
 #define COLOR_INPUT_ENABLED "#D3B5B5"
 
-#define COLOR_WHITE            "#FFFFFF"
+#define COLOR_WHITE            "#EEEEEE"
 #define COLOR_SILVER           "#C0C0C0"
 #define COLOR_GRAY             "#808080"
-#define COLOR_FLOORTILE_GRAY   "#8D8C8C"
+#define COLOR_FLOORTILE_GRAY   "#8D8B8B"
 #define COLOR_ALMOST_BLACK	   "#333333"
 #define COLOR_BLACK            "#000000"
 #define COLOR_RED              "#FF0000"
 #define COLOR_RED_LIGHT        "#FF3333"
 #define COLOR_MAROON           "#800000"
 #define COLOR_YELLOW           "#FFFF00"
 #define COLOR_OLIVE            "#808000"
-#define COLOR_LIME             "#00FF00"
+#define COLOR_LIME             "#32CD32"
 #define COLOR_GREEN            "#008000"
 #define COLOR_CYAN             "#00FFFF"
 #define COLOR_TEAL             "#008080"
 #define COLOR_BLUE             "#0000FF"
 #define COLOR_BLUE_LIGHT       "#33CCFF"
 #define COLOR_NAVY             "#000080"
-#define COLOR_PINK             "#FF00FF"
+#define COLOR_PINK             "#FFC0CB"
+#define COLOR_MAGENTA          "#FF00FF"
 #define COLOR_PURPLE           "#800080"
 #define COLOR_ORANGE           "#FF9900"
 #define COLOR_BEIGE            "#CEB689"
-#define COLOR_BLUE_GRAY        "#6A97B0"
-#define COLOR_BROWN            "#B19664"
-#define COLOR_DARK_BROWN       "#917448"
-#define COLOR_DARK_ORANGE      "#B95A00"
-#define COLOR_GREEN_GRAY       "#8DAF6A"
-#define COLOR_RED_GRAY         "#AA5F61"
-#define COLOR_PALE_BLUE_GRAY   "#8BBBD5"
-#define COLOR_PALE_GREEN_GRAY  "#AED18B"
-#define COLOR_PALE_RED_GRAY    "#CC9090"
-#define COLOR_PALE_PURPLE_GRAY "#BDA2BA"
-#define COLOR_PURPLE_GRAY      "#A2819E"
+#define COLOR_BLUE_GRAY        "#75A2BB"
+#define COLOR_BROWN            "#BA9F6D"
+#define COLOR_DARK_BROWN       "#997C4F"
+#define COLOR_DARK_ORANGE      "#C3630C"
+#define COLOR_GREEN_GRAY       "#99bb76"
```

Comment:
Capitalization is inconsistent with the rest of the defines here.

Literally unplayable